--- a/backend/lib/config.js
+++ b/backend/lib/config.js
@@ -7,7 +7,7 @@
-const keysFile = '/data/keys.json';
+const keysFile = '/config/keys.json';
 
--- a/backend/internal/nginx.js
+++ b/backend/internal/nginx.js
@@ -126,9 +126,9 @@
 	getConfigName: (host_type, host_id) => {
 		if (host_type === 'default') {
-			return '/data/nginx/default_host/site.conf';
+			return '/config/nginx/default_host/site.conf';
 		}
-		return '/data/nginx/' + internalNginx.getFileFriendlyHostType(host_type) + '/' + host_id + '.conf';
+		return '/config/nginx/' + internalNginx.getFileFriendlyHostType(host_type) + '/' + host_id + '.conf';
 	},